packages/cli/commands/project/migrateApp.js

```diff
@@ -133,10 +138,19 @@ exports.handler = async options => {
       text: i18n(`${i18nKey}.migrationStatus.inProgress`),
     });
 
+    handleKeypress(async key => {
+      if ((key.ctrl && key.name === 'c') || key.name === 'q') {
+        SpinniesManager.remove('migrateApp');
+        logger.log(i18n(`${i18nKey}.migrationInterrupted`));
+        process.exit();
```

Comment:
Should we use `EXIT_CODES.SUCCESS` here?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1095/commits/26c2d72b52322f762d78188eadc2184cdb5ed7d2